test/index.js

```diff
@@ -742,6 +747,8 @@ describe('PaymentProtocol', function() {
         verifier.verify(pem, sig).should.equal(true);
       }
 
+      var clock = sandbox.useFakeTimers(1459105693843);
```

Comment:
we should make a comment showing what this date actually is. '2016-03-27T19:08:13.843Z'